node-x509.d.ts

```diff
@@ -0,0 +1,9 @@
+declare namespace x509 {
+  function getIssuer(pathOrPEM: string): any
+  function getAltNames(pathOrPEM: string): any
+  function getSubject(pathOrPEM: string): any
+  function parseCert(pathOrPEM: string): any
+  function verify(certPath: string, CACertChainPath: string): any
+}
+
+export = x509;
```

Comment:
Could you please add this in another pull-request, this is not relevant about the subject of this pull-request.